src/grpcbox.app.src

```diff
@@ -9,7 +9,7 @@
                  acceptor_pool,
                  gproc,
                  ctx]},
-  {env, [{client, #{channels => [%% {default_channel, round_robin, [{http, "localhost", 8080, []}], #{}}
```

Comment:
Fixed this example, the round_robin option should have been inside the last argument (`#{}`) but it is the default value anyway so just removed it.